docs/Project-Board-Guide.md

```diff
@@ -17,13 +17,13 @@ Please familiarise yourself with our:
 <!-- prettier-ignore -->
 | Column Name               | Issues/Pull Requests that belong in this column | When to move card to the next column | Additional context |
 | ------------------------- | ------------------------------------------------| ------------------------------------ |------------------- |
-| To-do | Contains issues that have been newly created and added to the Simorgh project. | Once the issue has been refined to the corresponding team that has been labelled to the issue, then it can be manually moved to "Ready for Dev". | n/a |
+| To-do | Contains issues that have been newly created and added to the Simorgh project. | Once the issue has been refined by the corresponding labelled team, then it can be manually moved to "Ready for Dev". | n/a |
```

Comment:
```suggestion
| To-do | Contains issues that have been newly created and added to the Simorgh project. | Once the issue has been refined by the corresponding labelled pod (aka team), then it can be manually moved to "Ready for Dev". | n/a |
```